content/en/news/releases/1.17.x/announcing-1.17.6/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Announcing Istio 1.17.6
+linktitle: 1.17.6
+subtitle: Patch Release
+description: Istio 1.17.6 patch release.
+publishdate: 2023-09-19
+release: 1.17.6
+---
+
+This release contains bug fixes to improve robustness. This release note describes what is different between Istio 1.17.5 and Istio 1.17.6.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** SELinux issue on CentOS9/RHEL9 where iptables-restore isn't allowed to open files in `/tmp`. Rules passed to iptables-restore are no longer written to a file, but are passed via stdin. ([Issue #42485](https://github.com/istio/istio/issues/42485))
+
+- **Fixed** an issue where Istio should be using IMDSv2 as possible on AWS. ([Issue #45825](https://github.com/istio/istio/issues/45825))
```

Comment:
Can remove from spelling since backticks?
```suggestion
- **Fixed** an issue that Istio should prefer `IMDSv2` on AWS. ([Issue #45825](https://github.com/istio/istio/issues/45825))
```